module-system/module-implementations/examples/sov-election/src/call.rs

```diff
@@ -8,7 +8,12 @@ use sov_modules_api::{CallResponse, Context};
 use sov_state::WorkingSet;
 
 /// Call actions supported byte the module.
-#[derive(borsh::BorshDeserialize, borsh::BorshSerialize, Debug, PartialEq)]
+#[cfg_attr(
+    feature = "native",
+    derive(serde::Serialize),
+    derive(serde::Deserialize)
+)]
```

Comment:
This annotation seems really easy to forget. Let's add a bound on the `CallMessage` associated type of the `Module` trait

Reply:
just discussed this with @bkolad there are some unintended consequences where the compiler complains and requires even Context/C to have the serde bounds. this is defintiely fixable, but the change seems a bit more involved, so created another issue to track it
[cli_parser](https://github.com/Sovereign-Labs/sovereign-sdk/issues/378)